controllers/usersignup/usersignup_controller.go

```diff
@@ -346,7 +346,9 @@ func (r *Reconciler) checkIfMurAlreadyExists(reqLogger logr.Logger, config toolc
 		if shouldManageSpace(userSignup) {
 			space, created, err := r.ensureSpace(reqLogger, userSignup, mur, spaceTier)
 			// if there was an error or the space was created then return to complete the reconcile, another reconcile will occur when space is created since this controller watches spaces
-			if err != nil || created {
+			if created {
+				return true, nil
+			} else if err != nil {
```

Comment:
I think we should check if there was an error first and then check if created.

Reply:
Good point! done in f19b8c1